jupyterhub/jupyterhub/base/traefik-serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: traefik-proxy
+  namespace: $(namespace)
```

Comment:
You don't need to put `namespace` here - it'll be deployed to the same NS as the rest of the resources